### DISCUSSION TITLE
Bug Report [about npm unpublish]

### DISCUSSION THREAD

**Discussion by Jiasm** [npm/feedback#142] [CLI] [answered]
Hi, there:

I found that some operations in the NPM package did not meet the expected results.
Below is an example process:

```bash
> npm init
> npm version 1.0.0
> npm publish
> npm version 1.1.0
> npm publish
> npm version 2.0.0
> npm publish --tag test
> npm version 2.1.0
> npm publish --tag test
> npm info pkg version # 1.1.0 ✅
> npm info pkg@test version # 2.1.0 ✅
> npm unpublish pkg@1.1.0
> npm info pkg version # 2.1.0 ❌
```

When I unpublished the latest version under the latest tag, the latest version became the latest version under other tags.

![image](https://user-images.githubusercontent.com/9568094/103867517-7767ae00-5102-11eb-82d0-e8e0c376fe11.png)

Looking forward to your reply.

**Comment by ethomson** [comment]
Indeed this does not appear to be the optimal behavior.  In the meantime, please update your `latest` dist-tag manually.

**Comment by MylesBorins** [comment, marked as answer]
If this is still an issue a bug should be opened at https://github.com/npm/cli/issues
